grappelli_safe/templates/admin/edit_inline/stacked.html

```diff
@@ -47,3 +48,34 @@
 <div class="sortablehelper">
     <h3><b>Sortable Helper</b></h3>
 </div>
+
+<script type="text/javascript">
+
+(function($) {
+  var prefix = "{{ inline_admin_formset.formset.prefix|escapejs }}";
+  $("#{{ inline_admin_formset.formset.prefix|escapejs }}-group .inline-related").stackedFormset({
+    prefix: prefix,
+    addText: "{% filter escapejs %}{% blocktrans with inline_admin_formset.opts.verbose_name|capfirst as verbose_name %}Add another {{ verbose_name }}{% endblocktrans %}{% endfilter %}",
+    deleteText: "{% filter escapejs %}{% trans 'Delete Item' %}{% endfilter %}",
+  });
+  $(document).on("formset:added", function(event, $row, targetPrefix) {
+    if (targetPrefix === prefix) {
+        // Tweak the Delete link to resemble the ones defined in template
+        var $deleteLink = $row.find(".inline-deletelink");
+        $deleteLink
+            .appendTo($row.find("ul.inline-item-tools"))
+            .wrap("<li/>")
+            .addClass("deletelink")
+            .attr("title", $deleteLink.text())
+            .text("");
+
+        // Make sure the __prefix__ placeholder is parsed in links
```

Comment:
AFAIK it's necessary only if the inlines contain `FileField` instances, as described in #81. Maybe include a link to that issue?